tools/autograd/gen_variable_factories.py

```diff
@@ -64,7 +64,10 @@ def process_function(decl, has_tensor_options, disable_autograd):
     requires_grad = "options.requires_grad()" if has_tensor_options else "false"
     if decl['name'].endswith('_like') and not has_tensor_options:
         # it's a tensor
-        actuals.append('{}.options().is_variable(false)'.format(actuals[0]))
+        if decl['name'] == 'empty_like':
```

Comment:
Umm... this looks very non-general. (I assume it gets fixed later in the stack but it is very shocking here)

Reply:
Yes, need to list by function until I convert all of them